ProjectToTest/User.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Reflection;
+
+namespace ProjectToTest
+{
+    internal class User
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public int Age { get; set; }
+        public User(string firstName, string lastName, int age)
```

Comment:
The question:
1) First we initialize properties via constructor.
2) Then we initialize properties via object initializer (the parenthesises `{}`).

Why? Why we need two times setup properties? Maybe I lost something???